Source/WebKit/UIProcess/RemoteLayerTree/mac/RemoteLayerTreeEventDispatcher.cpp

```diff
@@ -387,6 +400,17 @@ void RemoteLayerTreeEventDispatcher::stopDisplayLinkObserver()
     m_displayRefreshObserverID = { };
 }
 
+void RemoteLayerTreeEventDispatcher::removeDisplayLinkClient()
+{
+    auto* displayLink = this->existingDisplayLink();
```

Comment:
No need for `this->`

Reply:
ops, I was using existingDisplayLink as a name before. Will remove, thanks.